api/test/integration/test_image_integration.py

```diff
@@ -0,0 +1,70 @@
+"""
+End-to-end API tests for images.
+
```

Comment:
Should we also add something like "There are more tests for images in the `test_media_integration`"?

Reply:
Agree, added in 217dc28.